### DISCUSSION TITLE
阅读小说时，纵向滑动不支持高刷新率

### DISCUSSION THREAD

**Discussion by HiRA017** [xiaoyaocz/flutter_dmzj#131] [General]
如题

**Comment by xiaoyaocz** [comment]
你是什么机型呢，我在小米手机上测试始终是120hz

https://user-images.githubusercontent.com/16559082/226495356-635b01c0-b441-4910-82ff-a2ed884b5b37.mp4

**Reply by HiRA017** [reply]
是 Google Pixel 5
刷新率确实一直在峰值，但文字滚动的动画看起来十分不流畅（图片滚动就没这个问题

**Comment by xiaoyaocz** [comment]
em...我没测出啥区别，这个问题有空再看看
